addons/product/i18n/product.pot

```diff
@@ -32,6 +32,12 @@ msgstr ""
 msgid "%(percentage)s %% discount and %(price)s surcharge"
 msgstr ""
 
+#. module: product
+#: code:addons/product/models/product_pricelist.py:0
+#, python-format
+msgid "You cannot delete this pricelist (%s), it is used in other(s) pricelist(s) : \n%s"
+msgstr ""
+
```

Comment:
The translated strings should be sorted alphabetically, this new entry should go at its rights place in the file.